content/events/t1/repeat/compcafe.md

```diff
@@ -10,11 +10,11 @@ end_time = "21:00"
 location = "CS0.06"
 
 icon = "ph-coffee"
-colour = "academic"
+colour = "compcafe"
 +++
 
 𝚢𝚘𝚞 𝚙𝚛𝚘𝚋𝚊𝚋𝚕𝚢 𝚑𝚊𝚟𝚎 𝚜𝚘𝚖𝚎 𝚠𝚘𝚛𝚔, 𝚜𝚘 𝚠𝚑𝚢 𝚗𝚘𝚝 𝚍𝚘 𝚒𝚝 𝚠𝚒𝚝𝚑 𝚏𝚛𝚎𝚎 𝚜𝚗𝚊𝚌𝚔𝚜!
 
 Computing Cafe exists to help you with any assignments you have! There will be tons of people from different years and differents courses, so you'll find whoever you need.
 
-There is also free coffee from the department vending machine in the common room, so take advantage of that while you can :D
+There is also free coffee from the department vending machine in the common room, so take advantage of that while you can :D
```

Comment:
maybe say theres also tea and hot chocolate?

Reply:
The contents of the event descriptions isn't my business, you lot should probably tweak them anyway, unless you want serene's descriptions immortalized forever